modules/appeals_api/spec/docs/shared_docs_examples.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require Rails.root.join('spec', 'rswag_override.rb').to_s
+require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
+require AppealsApi::Engine.root.join('spec', 'support', 'doc_helpers.rb')
+
+shared_examples 'docs request' do |opts = {}|
+  include DocHelpers
+  let(:status_code) { opts[:status_code] || 200 }
+  let(:cassette_name) { opts[:cassette_name] }
+  let(:oauth_scopes) { opts[:oauth_scopes] || %w[] }
+  # Set the `multiple` option to true wherever there are multiple examples for a single response code. Otherwise, only
+  # the last example will be included in the generated docs.
+  let(:multiple) { opts[:multiple] || false }
```

Comment:
Setup and teardown for examples throughout the rswag specs seem to be really consistent - this shared example seems like it will be usable for almost all cases

Reply:
Hmm I thought I'd extracted some of this to other shared examples already, but maybe that was for slightly different scenarios' boilerplate stuff. IIRC what I wrote didn't cover all cases, just a decent number of them, so it's entirely possible what you've written shores that gap up 😄 Still, might want to look around (if you haven't already) to see if this isn't duplicated work.

Reply:
Thank you, I totally missed the existing shared example!! I refactored this to use that, and added a `cassette` option, which seemed helpful